java/1.2/modules/ROOT/pages/configure-java-module-acb-edit.adoc

```diff
@@ -0,0 +1,331 @@
+= Configure the Java Module Using Anypoint Code Builder
+
+Use the Java module in your Anypoint Code Builder project to define custom Java code. To use Anypoint Code Builder to configure the Java module, create a basic integration project in Anypoint Code Builder, add connectors and other components to your Mule application to process your business logic, and then configure the attributes for the Java Module. 
+
+== Before You Begin
+
+Before creating an Anypoint Code Builder integration project, you must:
+
+* xref:anypoint-code-builder::start-acb.adoc[Set up and access the Anypoint Code Builder web or desktop IDE]
+* Have credentials to access Anypoint Platform
+* Ensure that the Extension Pack for Java (from Microsoft) is installed in Visual Studio Code.
+
+[[creating-new-integration-project]]
+== Creating a New Integration Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-create-integration-project"]
+
+When you create a new Anypoint Code Builder project, `src/main/java` and `/src/test/java` folders are created in the project. For existing projects, you can manually create these folders. 
+
+[[adding-components-to-project]]
+== Adding Components to Your Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-add-components-to-project"]
+
+[[configuring-component-attributes]]
+== Configuring the Component Attributes
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-configure-attributes"]
+
+[[configure-trigger]]
+== Configure a Trigger for the Flow
+A trigger (source) starts the flow when specified conditions are met. If a connector has its own associated sources, you can use a connector-specific trigger to initiate the flow. Each connector trigger has a unique configuration.
+
+You can configure the following sources for this connector:
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+[[configuring-new]]
+=== Configuring the New Operation
+
+Create a new class instance by using the *New* operation of the Java module. The identifier of a constructor includes the class and constructor names, and the constructor has a full description of its signature, including the types of each parameter.
+
+For example, if you invoke the constructor `Me(String name, int age)`, which belongs to the `org.bar.Me` class, then the identifier of the method is `Me(String,int)`.
+
+To configure the attributes for *New* in the Anypoint Code Builder UI:
+
+. Click the *New* component on the canvas.
+. In the *General* tab:
+.. In *Class*, search and select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Constructor*, search and select the constructor to use. 
++
+NOTE: If this is blank, click the refresh icon to populate it.
```

Comment:
This field? 

https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22it,%22%22this,%22%22these%22